sql/src/test/java/io/crate/integrationtests/ShardStatsTest.java

```diff
@@ -82,6 +83,32 @@ public void testSelectGroupByIncludingUnassignedShards() throws Exception {
     }
 
     @Test
+    public void testSelectCountIncludingUnassignedShards() throws Exception {
+        execute("create table locations (id integer primary key, name string) with(number_of_replicas=2)");
+        ensureYellow();
+
+        execute("select count(*) from sys.shards where schema_name='doc' AND table_name='locations'");
+        assertThat(response.rowCount(), is(1L));
+        assertThat((Long) response.rows()[0][0], is(15L));
+    }
+
+    @Test
+    public void testOnlyUnassignedShards() throws Exception {
+        execute("set global transient cluster.routing.allocation.enable=none");
```

Comment:
this test runs in a global suite, so imho cluster settings could influence other tests